bootstrap/bootstrap.ps1

```diff
@@ -67,7 +67,10 @@ if (-Not (Test-Path $junctionParent))
 {
     mkdir $junctionParent | Out-Null
 }
-cmd.exe /c mklink /j $SharedFrameworkSymlinkPath $junctionTarget | Out-Null
+if (-Not (Test-Path $SharedFrameworkSymlinkPath))
```

Comment:
Are there cases were this symlink already exists but points to the wrong junction? I would think that is possible if it already exists. I wonder if we should always recreate?


Are there scenarios were we call this script multiple times? I thought the semaphore would block us from calling it multiple times.